### PR TITLE
Salesforce: remove sensitive fields in log

### LINKED ISSUE #3670: Salesforce connector exposing sensitive fields in logs
## Description
A user reported that the Salesforce connector is logging refresh tokens and client secret key. The connector should never output any sensitive info in logs like this. 

## Current Behavior
_Tell us what happens instead of the expected behavior._

## Logs
_If applicable, please upload the logs from the failing operation. For sync jobs, you can download the full logs from the UI by going to the sync attempt page and clicking the download logs button at the top right of the logs display window._

## Steps to Reproduce
1. 
1. 
1. 

## Severity of the bug for you
Very low / Low / Medium / High / Critical

## Airbyte Version
Found in the .env file in the root of the project.

## Connector Version (if applicable)
Found in the admin page in the UI in the respective Source or Destination tab.

## Additional context
_Environment, version, integration..._

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200368060087111/1200393319979788) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by yevhenii-ldv**
## What
*closes #3670.*

## How
*Describe the solution*

## Pre-merge Checklist
- [ ] *Run integration tests*
- [ ] *Publish Docker images*

## Recommended reading order
1. `test.java`
1. `component.ts`
1. the rest

### CODE REVIEW

**Comment by yevhenii-ldv** [comment]
Tests are disabled for this connector

**Review by vitaliizazmic — APPROVED** [review]

**Review by keu — APPROVED** [review]

**Review by sherifnada — APPROVED** [review]

**Comment by sherifnada** [comment]
@yevhenii-ldv can you uncomment the block in `build.gradle` then bump and publish?

File: airbyte-integrations/connectors/source-salesforce-singer/Dockerfile [hunk 1]
```diff
@@ -1,14 +1,15 @@
 FROM airbyte/integration-base-singer:0.1.1
 
-RUN apt-get update && apt-get install -y \
-    bash \
+RUN apt-get update \
+    && apt-get install -y bash \
+    && apt-get install -y gcc \
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-salesforce-singer/Dockerfile:5]: why was gcc needed?

Comment (yevhenii-ldv) [reply airbyte-integrations/connectors/source-salesforce-singer/Dockerfile:5]: We have an error when run CI tests

**Pull request merged**